SConstruct

```diff
@@ -1014,7 +1017,9 @@ if env["vsproj"]:
 if env["compiledb"]:
     if env.scons_version < (4, 0, 0):
         # Generating the compilation DB (`compile_commands.json`) requires SCons 4.0.0 or later.
-        print_error("The `compiledb=yes` option requires SCons 4.0 or later, but your version is %s." % scons_raw_version)
+        print_error(
```

Comment:
there isn't any change here, revert this.

Reply:
This was cherry picked, and that change was made on Godot's side. There are other build script changes that we may cherry pick, so bringing all changes over reduces the chances of a pointless merge conflict later on.